pkg/zk/generators.go

```diff
@@ -224,6 +230,22 @@ func makeZkEnvConfigString(z *v1beta1.ZookeeperCluster) string {
 }
 
 func makeService(name string, ports []v1.ServicePort, clusterIP bool, z *v1beta1.ZookeeperCluster) *v1.Service {
```

Comment:
Could we give a better name to the bool variable - ClusterIP. Perhaps it would be nice to call it `enableDNS`???

Reply:
This is not needed any more.